### PR TITLE
Add basics of programming lesson

### PR DESCRIPTION
**Pull request by lucywyman**
TODO:
* Add more pictures
* Write exercise (github.com/devopsbootcamp/bootcamp-exercises?)
* Anything missing?

### CODE REVIEW

File: source/bootcamp_content/python-programming-basics.rst [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+Try it out
+----------
+
+Open a repl (read eval print loop):
+
+.. code-block:: python
+
+    $ python
+    >>> name = # Your name
```
Comment (dspt) [inline source/bootcamp_content/python-programming-basics.rst]: Is this the common notation being used for user input variables?

File: source/bootcamp_content/python-programming-basics.rst [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+    >>> print name + " is " + str(age)
+
+* We need to convert age from int to string so it can print!
+
+Control flow
+------------
+
+.. code-block:: python
+
+    if name == "Lucy":
```
Comment (dspt) [inline source/bootcamp_content/python-programming-basics.rst]: == comparator might want to be mentioned as different than assignment

**Comment by dspt** [comment]
Bike shedded some things, but looks good to present :+1:

**Comment by ElijahCaine** [comment]
I already touched anything that I thought needed touching. The changes since then are good, :+1:

**Comment by dspt** [comment]
Builds :+1:

**Pull request merged**